From a04db15f5a8fff9d6e737c7a832cb81131a248c9 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Thu, 10 Mar 2016 18:18:32 +0800
Subject: [PATCH] ARM64: dts: rockchip: enable the tsadc for rk3399 chrome

This patch enables the tsadc for rk3399 evb board.

The rk3399 evb board uses the gpio to reset the chip since it connects the
PMIC to work, and TSHUT is low active on evb board.

Change-Id: Ibd4fc2c752fe1f34cc231385ee314e4b9a32e970
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-chrome.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts b/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts
index 62f009f62a5f..4ef705663bcb 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts
@@ -61,6 +61,12 @@
 	status = "okay";
 };
 
+&tsadc {
+	rockchip,hw-tshut-mode = <1>; /* tshut mode 0:CRU 1:GPIO */
+	rockchip,hw-tshut-polarity = <0>; /* tshut polarity 0:LOW 1:HIGH */
+	status = "okay";
+};
+
 &uart2 {
 	status = "okay";
 };
-- 
2.35.3

